site/docs/id/plugins/parse-mode.md

```diff
@@ -82,37 +188,35 @@ bot.start();
  <CodeGroupItem title="Deno">
 
 ```ts
-import { Bot } from "https://deno.land/x/grammy/mod.ts";
+import { Bot, Context } from "https://deno.land/x/grammy/mod.ts";
 import {
   hydrateReply,
   parseMode,
 } from "https://deno.land/x/grammy_parse_mode/mod.ts";
 
-import type { ParseModeContext } from "https://deno.land/x/grammy_parse_mode/mod.ts";
+import type { ParseModeFlavor } from "https://deno.land/x/grammy_parse_mode/mod.ts";
 
-const bot = new Bot<ParseModeContext>("");
+const bot = new Bot<ParseModeFlavor<Context>>("");
 
-// Gunakan plugin-nya
+// Install plugin-nya.
 bot.use(hydrateReply);
 
-// Atur setting-an bawaan parse_mode untuk ctx.reply
+// Atur parse_mode bawaan untuk ctx.reply
 bot.api.config.use(parseMode("MarkdownV2"));
 
 bot.command("demo", async (ctx) => {
-  await ctx.reply(
-    "*Reply* ini menggunakan _MarkdownV2_ sebagai `format` bawaannya",
-  );
+  await ctx.reply("*Teks* ini _diformat_ menggunakan `format bawaan`");
   await ctx.replyWithHTML(
-    "<b>Reply</b> ini menggunakan <code>format</code> <i>withHTML</i>",
+    "<b>Teks</b> ini <i>diformat</i> menggunakan <code>withHTML</code>",
   );
   await ctx.replyWithMarkdown(
-    "*Reply* ini menggunakan `format` _withMarkdown_",
+    "*Teks* ini _diformat_ menggunakan `withMarkdown`",
   );
   await ctx.replyWithMarkdownV1(
-    "*Reply* ini menggunakan `format` _withMarkdownV1_",
+    "*Teks* ini _diformat_ menggunakan `withMarkdownV1`",
```

Comment:
I was just wondering if markdownv1 and regular markdown are the same thing. I couldn't find any documentation for it on https://core.telegram.org/bots/api. All I found was documentation for v2. Not a big deal though, just curious.

Reply:
@roj1512 @KnorpelSenf

Reply:
Okay :')

Reply:
Thanks!